asf_tools/hand/prepare.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Generator, List, Union
+
+import shapely.geometry
+from osgeo import gdal, ogr
+
+from asf_tools.dem import GDALConfigManager, shift_for_antimeridian
+
+HAND_GEOJSON = '/vsicurl/https://asf-hand-data.s3.amazonaws.com/cop30-hand.geojson'
+
+gdal.UseExceptions()
+ogr.UseExceptions()
+
+
+def get_hand_features() -> Generator[ogr.Feature, None, None]:
+    ds = ogr.Open(HAND_GEOJSON)
+    layer = ds.GetLayer()
+    for feature in layer:
+        yield feature
+    del ds
+
+
+def intersects_hand(geometry: ogr.Geometry) -> bool:
+    for feature in get_hand_features():
+        if feature.GetGeometryRef().Intersects(geometry):
+            return True
+
+
+def get_hand_file_paths(geometry: ogr.Geometry) -> List[str]:
+    file_paths = []
+    for feature in get_hand_features():
+        if feature.GetGeometryRef().Intersects(geometry):
+            file_paths.append(feature.GetField('file_path'))
+    return file_paths
```

Comment:
These are all mostly copy-pastes from the `asf_tools.dem` and could be generalized for both -- I just wasn't sure where to put them... `util.py`? A new `vector.py`?